.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "automatic"
+}
```

Comment:
This looks like an editor-specific file - I don't think we want to check it in to the project.

Reply:
Whoops, I removed that now